### PR TITLE
Add scripts to set the environment up to use just-built artifacts

### PR DESCRIPTION
**Pull request by barcharcraz**
These are intended for development,
because I kept forgetting to add in the arch prefix

This is a github-only change, there's nothing that will touch the internal build, and
the files are copied over outside the "output" directory.

### CODE REVIEW

**Review by CaseyCarter — APPROVED** [review]

File: stl/set_environment.bat.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@set PATH="%~dp0out\bin\@VCLIBS_I386_OR_AMD64@;%PATH%"
```
Comment (StephanTLavavej) [inline stl/set_environment.bat.in:1]: Can you add copyright/license banners to these files? For `bat`, `::` works for comments.

File: stl/set_environment.bat.in [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@set PATH="%~dp0out\bin\@VCLIBS_I386_OR_AMD64@;%PATH%"
+@set INCLUDE="%~dp0out\inc;%INCLUDE%"
+@set LIB="%~dp0\out\lib\@VCLIBS_I386_OR_AMD64@;%LIB%"
```
Comment (StephanTLavavej) [inline stl/set_environment.bat.in:3]: Unlike `%~dp0out` above, `%~dp0\out` adds an extra backslash here.

**Review by StephanTLavavej — CHANGES_REQUESTED** [review]
Thanks - these scripts will make life more convenient.

**Review by StephanTLavavej — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
Verified that both the BAT and PS1 produce the same paths that my local incantations do. FYI @CaseyCarter as significant changes were pushed.

**Comment by StephanTLavavej** [comment]
I'm mirroring this to the MSVC-internal repo - please notify me if any further changes are pushed.

**Comment by StephanTLavavej** [comment]
Thanks again for this quality-of-life improvement! :smile_cat: :tada: :rocket:

**Pull request merged**